tests/post/steps/test_service_configuration.py

```diff
@@ -92,30 +78,26 @@ def update_service_configuration(
     value
 ):
 
-    csc_from_yaml = yaml.safe_load(
-        DEFAULT_CSC_CONFIG_YAML.format(replicas=value)
-    )
+    csc_obj = csc.get(name, namespace)
+    csc_content = csc.load(csc_obj, name, namespace)
```

Comment:
Sorry it's probably just existing code, but what are those two methods doing? Why not just have one?

Reply:
Yes, it's already existing code and I agree that it should be done in the same method.
I wanted to change this, but I forgot. Will do it.